charts/lnd/templates/NOTES.txt

```diff
@@ -21,9 +21,14 @@
   kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:$CONTAINER_PORT
 {{- end }}
 2. To get the TLS, run:
-export TLS=$(kubectl -n {{ .Release.Namespace }} exec {{ .Release.Name }}-0 -- base64 /root/.lnd/tls.cert | tr -d '\n\r')
+export TLS=$(kubectl -n {{ .Release.Namespace }} exec {{ include "lnd.fullname" . }}-0 -- base64 /root/.lnd/tls.cert | tr -d '\n\r')
 3. To get the macaroon, run:
-export MACAROON=$(kubectl exec -n {{ .Release.Namespace }} {{ .Release.Name }}-0 -- base64 /root/.lnd/data/chain/bitcoin/{{.Values.global.network}}/admin.macaroon | tr -d '\n\r')
+export MACAROON=$(kubectl exec -n {{ .Release.Namespace }} {{ include "lnd.fullname" . }}-0 -- base64 /root/.lnd/data/chain/bitcoin/{{.Values.global.network}}/admin.macaroon | tr -d '\n\r')
 4. To execute lncli against the pod, run the following commands:
-kubectl -n {{ .Release.Namespace }} port-forward {{ .Release.Name }}-0 10009
+kubectl -n {{ .Release.Namespace }} port-forward {{ include "lnd.fullname" . }}-0 10009
 lncli -n {{ .Release.Namespace }} help
+5. To retrieve the seed for the lnd wallet, run:
+kubectl -n {{ .Release.Namespace }} logs {{ include "lnd.fullname" . }}-wallet-create
+kubectl -n {{ .Release.Namespace }} delete pod {{ include "lnd.fullname" . }}-wallet-create
```

Comment:
why delete the pod after?

Reply:
To remove one trace of the seed, so that it's not very easy to retrieve it. Lncli does the same if I'm not wrong, meaning that once that screen goes away, you can't get it back.

Reply:
you can from the db but there is no api for it.

Reply:
Yes. Basically the idea is to not add a way to retrieve the seed easily. Do you feel otherwise?

Reply:
I agree. should be written offline somewhere as a backup

the way to get the seed then would be (for us or an attacker):
- get the secret related to the passwod
- get access to the disk to get the encrypted boltdb (encrypted with the secret password above)

I believe one way to improve the security quite significantly would be to integrate [Hashicorp Vault](https://www.vaultproject.io/docs). 

The idea would be that the secret would be stored in memory only but not accessible by us (or at least not as easily as a Kubernetes Secret). But this would require time to think about it and document ourselves to Vault, so this is for another PR!